### PR TITLE
Changed landing page to use swagger and included links to repository (Issue #15)

### PR DESCRIPTION
**Pull request by apham1**

### CODE REVIEW

**Review by kwurst — CHANGES_REQUESTED** [review]
@apham1 

This looks like you did not start from the most recent version of the code. The build.gradle file you are submitting in this commit is reverting the baseName, mainClassName, and version to their previous values.

Also, the version number should go up. The last committed version was 1.0.0. This should probably be version 1.1.0. (See http://semver.org for the rules.)

**Comment by apham1** [comment]
I accidentally overwrote those while copying over the added dependencies for swagger, thanks.

Fixed in commit 7e3569051bf76ba853716b03725e89360bf3f0e3. Version number set to 1.1.0

**Review by kwurst — CHANGES_REQUESTED** [review]
@apham1 
The API version needs to be updated in SwaggerConfig.java also

File: src/main/java/com/github/LibreFoodPantry/FoodKeeperAPI/Configs/SwaggerConfig.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	}
+	
+	private ApiInfo apiInfo() {
+		return new ApiInfoBuilder()
+				  .title("FoodKeeper API")
+				  .description("API for USDA FSIS FoodKeeper data (https://github.com/LibreFoodPantry/FoodKeeper-API)")
+				  .license("GNU General Public License v3.0")
+				  .licenseUrl("https://www.gnu.org/licenses/gpl-3.0.en.html")
+				  .termsOfServiceUrl("")
+				  .version("1.0.0")
```
Comment (kwurst) [inline src/main/java/com/github/LibreFoodPantry/FoodKeeperAPI/Configs/SwaggerConfig.java]: @apham1 The change needs to be made here.

**Review by kwurst — CHANGES_REQUESTED** [review]
There is a second place to change the version - in SwaggerConfig.java

File: src/main/java/com/github/LibreFoodPantry/FoodKeeperAPI/Configs/SwaggerConfig.java (hunk 1 above)
Comment (kwurst) [reply src/main/java/com/github/LibreFoodPantry/FoodKeeperAPI/Configs/SwaggerConfig.java]: This is what shows up in the green "bubble" on the HTML page.

Comment (apham1) [reply src/main/java/com/github/LibreFoodPantry/FoodKeeperAPI/Configs/SwaggerConfig.java]: Updated in 095e123

**Review by kwurst — APPROVED** [review]

**Pull request merged**